doc/man/man3/crypto_lock.3monocypher

```diff
@@ -218,8 +220,10 @@ size_t         text_size;    /* Message size (NOT secret) */
 uint8_t       *plain_text;   /* Secret message            */
 if (crypto_unlock(plain_text, key, nonce, mac,
                   cipher_text, text_size)) {
-    /* The message is corrupted */
-    /* Abort the decryption     */
+    /* The message is corrupted           */
+    /* Wipe key if it is no longer needed */
+    /* and abort the decryption           */
```

Comment:
I know I'm late since the original did it, but shouldn't we be doing this instead?

    /* The message is corrupted.
     * Wipe key if it is no longer needed
     * and abort the decryption.
     */

Applies to the next one, too.

Reply:
Correct, this is lighter that way. I should have done it like this from the start.